### ISSUE TITLE
Ignore external Jars (ext:"jar")

### ISSUE THREAD

**Issue by AvoMMH** [jk1/Gradle-License-Report#262]
Hey,
I have a project with some custom jars.
When I try to generate the report i get an error:
`
> Task :generateLicenseReport
Failed to retrieve artifacts for [group:, name:[name], version:, ext:pom]
org.gradle.internal.resolve.ArtifactNotFoundException: Could not find [name]-.pom (:[name]:).
Searched in the following locations:
    file:/C:/Project/lib/[name].pom
`
I tried excluding the dep manually with:
`
licenseReport {
    excludes = [":ojdbc8:"]
    renderers = [new InventoryHtmlReportRenderer()]
    filters = [new LicenseBundleNormalizer()]
}
`
(I also tried it without the double dot)

TLDR: I have a custom jar, that I want to exclude from the report.
